style.css

```diff
@@ -449,6 +449,84 @@ footer {
   display: none;
 }
 
+#detail {
+  background-color: #c1c7d0;
+  display: none;
+  position: fixed;
+  z-index: 9;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+}
+
+.detail {
+  border-radius: 16px;
+  background-color: #fff;
+  padding: 16px;
+  margin: 20px 16px 26px 16px;
+}
+
+.detail #mobile-img,
+#desktop-img {
+  width: 100%;
+  height: auto;
+}
+
+.detail #desktop-img {
+  display: none;
+}
+
+.detail #close {
+  position: absolute;
+  top: 8%;
+  right: 12%;
+  font-size: 25px;
+  color: #fff;
+  z-index: 99;
+}
+
+.detail #close:hover {
+  color: #091e42;
+}
+
+.detail .detail-caption .detail-title {
+  font-size: 32px;
+  font-weight: 600;
+  line-height: 44px;
+  color: #172b4d;
+  margin-top: 10px;
+}
+
+.detail .detail-caption .tag {
+  justify-content: flex-start;
+  gap: 8px;
+}
+
+.detail .detail-caption .tag .desktop-tag {
+  display: none;
+}
+
+.detail .detail-text {
+  color: #344563;
+  line-height: 24px;
+}
+
+.detail .detail-button {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 16px;
+}
+
+.detail .detail-button button {
+  padding: 12px 15px;
+}
+
+.detail .detail-button i {
+  margin-left: 10px;
+}
```

Comment:
In the mobile version, kindly ensure that the margins and dimensions of the `see source` and  `see live` buttons matches that of the figma since there are some slight differences at certain screen widths as shown below :point_down: 

### Your design

![Screenshot from 2022-12-21 06-47-25](https://user-images.githubusercontent.com/65233860/208840839-0b758e9a-5ae5-4a22-b393-9ba03e72d9e4.png)



### Figma design


![Screenshot from 2022-12-21 06-47-16](https://user-images.githubusercontent.com/65233860/208840876-1e86ddae-3657-49bd-8cf7-ed03aaf89501.png)